libp2p/src/builder.rs

```diff
@@ -113,7 +120,57 @@ mod tests {
     }
 
     #[test]
-    #[cfg(all(feature = "tokio", feature = "tcp", feature = "tls", feature = "mplex"))]
+    #[cfg(all(feature = "tokio", feature = "quic"))]
```

Comment:
I am wondering if we should have these cfg's at all. We run tests always with all features anyway.

Reply:
Should I remove them from all the test cases of this file?

Reply:
Lets merge this as is but it might be worth putting as a proposal to @mxinden.

Reply:
> Lets merge this as is but it might be worth putting as a proposal to @mxinden.

I am fine either way @oblique and @thomaseizinger.